nucliadb/nucliadb/ingest/orm/brain.py

```diff
@@ -354,31 +407,35 @@ def process_meta(
         field_key: str,
         metadata: FieldMetadata,
         tags: Dict[str, List[str]],
-        relationnodedocument: RelationNode,
+        relation_node_document: RelationNode,
+        user_canceled_labels: List[str],
     ):
         for classification in metadata.classifications:
-            tags["l"].append(f"{classification.labelset}/{classification.label}")
-            relationnodelabel = RelationNode(
-                value=f"{classification.labelset}/{classification.label}",
-                ntype=RelationNode.NodeType.LABEL,
-            )
-            self.brain.relations.append(
-                Relation(
-                    relation=Relation.ABOUT,
-                    source=relationnodedocument,
-                    to=relationnodelabel,
+            label = f"{classification.labelset}/{classification.label}"
+            if label not in user_canceled_labels:
+                tags["l"].append(label)
+                relationnodelabel = RelationNode(
```

Comment:
to be visually coherent with the context code `relation_node_label` would be better